crates/paimon/src/io/file_io.rs

```diff
@@ -72,6 +74,9 @@ impl FileIO {
 
         Ok(FileStatus {
             size: meta.content_length(),
+            is_dir: meta.is_dir(),
+            last_modification_time: meta.last_modified(),
```

Comment:
How about using `last_modified` or `modified` for short?

Reply:
done